### PR TITLE
literally just a typo in sample study

### PR DESCRIPTION
**Pull request by RoriCremer**

### CODE REVIEW

**Review by thathert — DISMISSED** [review]

File: src/test/resources/study-minimal.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
     ],
     "assets":        [
       {
-        "name":   "Sample",
+        "name":   "sample",
```
Comment (ddietterich) [inline src/test/resources/study-minimal.json:31]: That will break my dataset connected tests.
So either don't do this or make the corresponding change to the dataset test.

**Review by ddietterich — CHANGES_REQUESTED** [review]
Fixing that typo has consequences

**Review by thathert — APPROVED** [review]
LGTM

**Review by ddietterich — APPROVED** [review]
Think you caught them all. Thanks!

**Pull request merged**